news/2019-04-08-project-loved-week-of-april-7th.md

```diff
@@ -0,0 +1,225 @@
+
+### Navigation
+
+- [osu!](#osu)
+- [osu!taiko](#osutaiko)
+- [osu!catch](#osucatch)
+- [osu!mania](#osumania)
+
+## <a name="osu" id="osu"></a>osu!
+
+osu! Loved candidates were chosen by [DigitalHypno](https://osu.ppy.sh/users/4384207), [-Roxas](https://osu.ppy.sh/users/1986262), [BeasttrollMC](https://osu.ppy.sh/users/3171691), [Bubbleman](https://osu.ppy.sh/users/5182050) and [waywern2012](https://osu.ppy.sh/users/5870453)!
+
+---
+
+[![](/wiki/shared/news/2019-04-08-project-loved-week-of-april-7th/osu/hardware-store.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+["Weird Al" Yankovic - Hardware Store](https://osu.ppy.sh/beatmapsets/833927#osu) by [Mr Moseby](https://osu.ppy.sh/users/3792472)  
+126 BPM, 3:30 | 7.75★, 8.93★, 9.01★  
+*written by [BeasttrollMC](https://osu.ppy.sh/users/3171691)*
+
+Over time we've seen quite a few competitive standard players dipping their toes into the mapping scene. One of these players is Woey (now known as Mr Moseby), a name recognizable not only for his skill but for his success in the mapping scene as well. Today we will be featuring Woey's take on "Weird Al" Yankovic's "Hardware Store", a map that has been notoriously popular amongst top players ever since its release. The mapset has three difficulties: "Matching Salt And Pepper Shakers" being the complete version at just over 9 stars, "POG moment" as a practice difficulty, and "a u t o m a t i c  c i r c u m c i s o r s (CS0)" for a CS0 version of the top difficulty. With the map featuring constant 252 BPM jumps, it's no surprise that the ones to popularize this map include some of the fastest players in the game.
+
+[![](/wiki/shared/news/2019-04-08-project-loved-week-of-april-7th/osu/doki-doki.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+[Thaehan - Doki-Doki](https://osu.ppy.sh/beatmapsets/519054#osu) by [Mr HeliX](https://osu.ppy.sh/users/2330619), [MujouSekai](https://osu.ppy.sh/users/5412070), [Mechanizen](https://osu.ppy.sh/users/6168393) and [rockstarrzz](https://osu.ppy.sh/users/4221626)  
+180 BPM, 1:42 | 1.25★ – 6.32★  
+*written by [BeasttrollMC](https://osu.ppy.sh/users/3171691)*
+
+It's obvious that this song has been extremely popular among all players, especially after two separate mapsets were Loved in the past, but both were extremely difficult and left out less experienced players who had little chance at passing them. This mapset by Mr HeliX fixes that, featuring a full mapset with difficulties ranging from 1.25 to 6.32 stars. With guest difficulties from rockstarrzz, Mechanizen and MujouSekai, this mapset is meant to give players from all skill ranges a chance at enjoy the music and having a blast. With two mapsets already Loved, is this a good reason to add a third? You decide in this week's Project Loved! Check out the map don't forget to vote for its chance to be Loved among the rest.
```

Comment:
You decide -> You shall decide?
Check out the map don't forget -> Check out the map and don't forget

Reply:
think "you decide" is a pretty common expression to mean "you [can/may/shall] decide", fixed second part though